src/object_store/src/object/s3.rs

```diff
@@ -571,6 +568,30 @@ impl S3ObjectStore {
         Self::configure_bucket_lifecycle(&client, bucket.as_str())
             .await
             .unwrap();
+
+        // check whether use batch delete
+        let test_path = "test_path";
```

Comment:
Using the same path name across different CNs for the check can be problematic when there is a race. Consider the following sequence:
1. CN1 executes L573-L583
2. CN2 executes L573-L583
3. CN2 executes L584-L594
4. CN1 executes L584-L594

CN1 will see object_store_use_batch_delete == false and CN2 will see object_store_use_batch_delete == true. Note that this race can also happen across clusters if user uses the same bucket for different clusters.

There are 4 solutions:
1. Use a random path name
2. Use a CN specific path name
3. Check the batch delete error and filter out NotExists error
4. Let meta do the check and inform CNs

Also, I suggest we add a risingwave specific string or `data_directory` as the prefix for the path name to avoid mistakenly overwriting and deleting user data in their bucket.

Reply:
Good catch! Let's use random string to avoid race.